api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.6.x.xml

```diff
@@ -133,5 +133,15 @@
 		<comment>Adding not null constraint to encounter_diagnosis.dx_rank. TRUNK-6081</comment>
 		<addNotNullConstraint tableName="encounter_diagnosis" columnName="dx_rank" columnDataType="int"/>
 	</changeSet>
+
+	<changeSet id="TRUNK-6204-2023-10-23" author="manojll">
+		<preConditions onFail="MARK_RAN">
+			<not><indexExists indexName="location_tag_name"/></not>
```

Comment:
One last thing: it would be best to specify `tableName` here as well.

Reply:
@ibacher I made the change that you requested.